<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>device_hub</class>
 <widget class="QWidget" name="device_hub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>527</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Device Hub</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#device_hub { border: 0px }</string>
  </property>
  <layout class="QGridLayout" name="device_hub_layout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>2</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="device_list_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="device_list_buttons">
         <property name="styleSheet">
          <string notr="true">#device_list_buttons { border: 0px }</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="device_list_buttons_layout">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="refresh_devices_button">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resources.qrc">
              <normaloff>:/device_hub/refresh_devices_enabled</normaloff>
              <disabledoff>:/device_hub/refresh_devices_disabled</disabledoff>:/device_hub/refresh_devices_enabled</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="devices_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>450</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="device_list">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Device</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>Jennie</string>
          </property>
          <property name="icon">
           <iconset resource="../../resources/resources.qrc">
            <normaloff>:/device_hub/computer</normaloff>:/device_hub/computer</iconset>
          </property>
          <item>
           <property name="text">
            <string>Camera</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/resources.qrc">
             <normaloff>:/device_hub/camera</normaloff>:/device_hub/camera</iconset>
           </property>
           <item>
            <property name="text">
             <string>Hamamatsu C114400-22CU (8x0521)</string>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resources.qrc">
              <normaloff>:/device_hub/camera</normaloff>:/device_hub/camera</iconset>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>Light Source</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/resources.qrc">
             <normaloff>:/device_hub/lightsource</normaloff>:/device_hub/lightsource</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Modulator</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/resources.qrc">
             <normaloff>:/device_hub/modulator</normaloff>:/device_hub/modulator</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Motion</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/resources.qrc">
             <normaloff>:/device_hub/motion</normaloff>:/device_hub/motion</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sensor</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/resources.qrc">
             <normaloff>:/device_hub/sensor</normaloff>:/device_hub/sensor</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sequencer</string>
           </property>
           <property name="icon">
            <iconset resource="../../resources/resources.qrc">
             <normaloff>:/device_hub/sequencer</normaloff>:/device_hub/sequencer</iconset>
           </property>
          </item>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <layout class="QVBoxLayout" name="property_editor_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="property_editor_buttons">
         <property name="styleSheet">
          <string notr="true">#property_editor_buttons { border: 0px }</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="property_editor_buttons_layout">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="refresh_properties_button">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset resource="../../resources/resources.qrc">
              <normaloff>:/device_hub/refresh_properties_enabled</normaloff>
              <disabledoff>:/device_hub/refresh_properties_disabled</disabledoff>:/device_hub/refresh_properties_enabled</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="properties_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>260</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="ParameterTree" name="property_editor" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParameterTree</class>
   <extends>QWidget</extends>
   <header>pyqtgraph.parametertree</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
